bdb/info.c

```diff
@@ -1890,6 +1890,38 @@ void bdb_process_user_command(bdb_state_type *bdb_state, char *line, int lline,
         pgno = toknum(tok, ltok);
 
         __pgtrash(bdb_state->dbenv, fileid, pgno);
+    } else if (tokcmp(tok, ltok, "fill") == 0) {
+        double percent_free_pages, percent_full_pages;
+
+        tok = segtok(line, lline, &st, &ltok);
```

Comment:
We are not using this table name below and reporting fill-factor for all the tables. Typo?

Reply:
agree